docs/csharp/language-reference/compiler-messages/cs8515.md

```diff
@@ -0,0 +1,43 @@
+---
+description: "Compiler Error CS8515"
+title: "Compiler Error CS8515"
+ms.date: 24/10/2022
+f1_keywords:
+  - "CS8515"
+helpviewer_keywords:
+  - "CS8515"
+---
+# Compiler Error CS8515
+
+Parentheses are required around the switch governing expression.
+
+
+## Example
+
+ The following sample generates CS8515:
+
+```csharp
+// CS8515.cs
+
+int x = 5;
+switch (x)
+    case 5:
+        break;
+
+```
+
+## To correct this error
+
+Add curly brackets: `{`  `}`:
```

Comment:
Based on earlier comment:

```suggestion
Add parentheses: `(`  `)`:
```

Reply:
I see, well I will add this one too as an example. Without the { } you will get also the 8515 error message, but you get other errors too: cs0118, cs1525, cs0219,cs1514, cs1513 AND cs8515 - tested with .NET 6 - Console App.